frontend/its-our-call/src/App.tsx

```diff
@@ -13,6 +13,8 @@ class App extends React.Component {
           <Header />
           <Switch>
             <Route exact={true} path="/" component={pages.Home}/>
+            <Route exact={true} path="/issues" component={pages.IssueList}/>
```

Comment:
Are you planning on pulling the paths into functions in pages/urls.tsx as in the legislators list? I think either way is fine.